File: python/fibonacci.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  fibonacci.py



def czy_naturalna(n):
    if n.isdigit():
        return True
    return False


def main(args):
    n = input('Który wyraz ciągu? ')
    while not czy_naturalna(n):
        print('Błędne dane')
        n = input('Który wyraz ciągu? ')
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/fibonacci.cpp
/*
 * fibonacci.cpp
 * 
 * Copyright 2019  <>
 */


#include <iostream>
using namespace std;
int main(int argc, char **argv)
{
	int a=0;
    int b=1;
    int n=0;
    cout<<"Który wyraz ciągu? ";
    cin>>n;
    if(n==0){
        a=0;}
        else
    for(int i = 0;i<=n-1;i++){
        a=a+b;
        b=a-b;
        }
        cout<<a;
	return 0;
}

